src/cgame/etj_jump_speeds.cpp

```diff
@@ -0,0 +1,202 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2021 ETJump team <zero@etjump.com>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#include "etj_jump_speeds.h"
+#include "etj_utilities.h"
+
+namespace ETJump
+{
+	void DrawJumpSpeeds()
+	{
+		playerState_t* ps = getValidPlayerState();
+		float offX = 0.0f;
+		float offY = 0.0f;
+		float x1 = 6 + etj_jumpSpeedsX.value;
+		float x2 = 6 + 30 + etj_jumpSpeedsX.value;
+		float y1 = 240 + etj_jumpSpeedsY.value;
+		float y2 = 240 + 12 +etj_jumpSpeedsY.value;
+		bool drawShadow = etj_jumpSpeedsShadow.integer;
+		vec4_t color;
+		bool vertical = !etj_jumpSpeedsStyle.integer;
+
+		if (!etj_drawJumpSpeeds.integer)
```

Comment:
Consider putting this to the top of the function? Could also combine all the checks.
```cpp
if (!etj_drawJumpSpeeds.integer ||
    ps->persistant[PERS_TEAM] == TEAM_SPECTATOR ||
    cg.zoomedBinoc ||
    cg.zoomedScope)
{
    return;
}
```

Reply:
General code style defines variable for functions at the top like this, and I believe compiler will optimize it anyway. For those checks, I don't really like doing stuff like that when it checks for completely different things.